test/export_import_test.py

```diff
@@ -0,0 +1,40 @@
+import shutil, unittest, logging, os
+import config.root # Set source directory
+
+from unittest.mock import patch
+from subprocess import call
+
+log = logging.getLogger("Test Import Export")
+logging.basicConfig(format='%(asctime)s %(name)s:%(message)s', level=logging.DEBUG)
+
+class Test(unittest.TestCase):
+    @patch('logging.Logger.error')
+    def test_export_import_profile_data(self, mock_error):
+        # Get current working directory
+        current_dir = os.getcwd()
+        # Change working directory to extras to run script and export data
+        try:
+            os.chdir("../simulators/extras") # IDE
+        except:
+            os.chdir("simulators/extras") # Command line
+
```

Comment:
- delete the files in "export_data" to make sure we only see the newly exported data in the test